index.html

```diff
@@ -0,0 +1,271 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;500;600;700&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>Document</title>
+</head>
+<body>  
+    <div class="containerglav"> 
+    <div class="conteiner">
+    <div class="headerline">
```

Comment:
Seems like you have some mess in HTML layout here. 
YOu put all your code inside `.containerglav` and `container`. What for?

Reply:
I wanted the header image to have no borders. Therefore, I set the container to other values, and for the "conteinerglav" I set the image. I hope you underst
and